### PR TITLE
Reduce pql queries from 10k to 500 in pinot-integration-tests module

### PR DESCRIPTION
**Pull request by jackjlli**
## Description
This PR:
* removes all the queries that return empty result.
* reduces pql queries from 10k to 500 in pinot-integration-tests module.

### CODE REVIEW

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java [hunk 1]
```diff
@@ -320,19 +320,17 @@ public void testQueriesFromQueryFile()
...
-          reader.readLine();
-        }
-
         String queryString = reader.readLine();
         // Reach end of file.
         if (queryString == null) {
           return;
         }
+        // Skip commented line and empty line.
+        if (queryString.startsWith("#") || queryString.isEmpty()) {
```
Comment (sajjad-moradi) [inline pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java:331]: Use `queryString.trim()` to include leading whitespaces before # or empty lines with only whitespaces.

**Review by sajjad-moradi — APPROVED** [review]

**Review by mcvsubbu — COMMENTED** [review]
You may want to add a couple of queries that return empty results also.

**Comment by mayankshriv** [comment]
Do we have a coverage report for this one? Granted that queries with empty results are not effective, but if the code path is still exercised and we see a drop in coverage, we may want to substitute these with new queries that bring the coverage back up.

**Comment by jackjlli** [comment]
> You may want to add a couple of queries that return empty results also.

Not necessarily, the queries with empty response will skip the comparison of the results like below:
https://github.com/apache/incubator-pinot/blob/f2a990bdca52f2227b64e3629435b28e6a800f3a/pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java#L527

**Comment by mayankshriv** [comment]
@jackjlli Slightly tangential question, most users are either migrating to or already using SQL. Is there a SQL version these queries that we are validating against?

**Review by Jackie-Jiang — COMMENTED** [review]
Since we want to keep only 500 queries, we might want to decorate them:
1. Remove `TOP` and `LIMIT` from aggregation only queries
2. Remove queries with more than 10000 groups which won't be compared, and remove the `LIMIT 100` from the hsql queries
3. Add some selection queries

**Comment by jackjlli** [comment]
> @jackjlli Slightly tangential question, most users are either migrating to or already using SQL. Is there a SQL version these queries that we are validating against?

Yes, there will be a PR for SQL queries as well after this one.

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java [hunk 2]
```diff
@@ -320,19 +320,18 @@ public void testQueriesFromQueryFile()
     assertNotNull(resourceUrl);
     File queryFile = new File(resourceUrl.getFile());
 
-    int maxNumQueriesToSkipInQueryFile = getMaxNumQueriesToSkipInQueryFile();
     try (BufferedReader reader = new BufferedReader(new FileReader(queryFile))) {
       while (true) {
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java]: Updated.

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java [hunk 3]
```diff
@@ -60,7 +60,7 @@
 
   // Default settings
   private static final String DEFAULT_PQL_QUERY_FILE_NAME =
-      "On_Time_On_Time_Performance_2014_100k_subset.test_queries_10K";
+      "On_Time_On_Time_Performance_2014_100k_subset.test_queries_1K";
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java]: Changed.

**Comment by jackjlli** [comment]
@Jackie-Jiang Updated the pql query file.

**Review by Jackie-Jiang — APPROVED** [review]
LGTM. This is a good starting point, let's keep improving this query set for better coverage. Thanks for working on this.

**Pull request merged**